zmon_worker_monitor/main.py

```diff
@@ -82,22 +83,27 @@ def main(args=None):
         port = config.get('redis.port', 6379)
         config.update({"redis.servers": '{}:{}'.format(config["redis.host"], port)})
 
+    log_level = config.get('loglevel', 'INFO')
     # save config in our settings module
-    settings.set_workers_log_level(config.get('loglevel', 'INFO'))
+    settings.set_workers_log_level(log_level)
     settings.set_external_config(config)
     settings.set_rpc_server_port(config.get('server.port'))
 
     logging.config.dictConfig(settings.RPC_SERVER_CONF['LOGGING'])
 
     logger = logging.getLogger(__name__)
 
+    logger.info('Initializing opentracing tracer: {}'.format(args.opentracing))
```

Comment:
Kind of misleading here. I would move it to the parts where it is really initialized.